libs/libm/newlib/Kconfig

```diff
@@ -3,6 +3,14 @@
 # see the file kconfig-language.txt in the NuttX tools repository.
 #
 
+config LIBM_USE_NEWLIB_ESP32
+	bool "Use Espressif ported newlib-esp32"
+	depends on LIBM_NEWLIB
```

Comment:
remove, don't need in this Kconfig

Reply:
Updated. Directly used `CONFIG_ARCH_CHIP_ESP32` to decide what newlib package will be used.